chain/cosmos/chain_node.go

```diff
@@ -257,12 +262,29 @@ func (tn *ChainNode) maybeLogBlock(height int64) {
 	if len(txs) == 0 {
 		return
 	}
-	pp, err := tendermint.PrettyPrintTxs(ctx, txs, tn.Client.Tx)
-	if err != nil {
-		tn.logger().Info("Failed to pretty print block", zap.Error(err))
-		return
+	sb := new(strings.Builder)
+	sb.WriteString("BLOCK INFO\n")
+	sb.WriteString(fmt.Sprintf("BLOCK HEIGHT: %d\n", height))
+	sb.WriteString(fmt.Sprintf("TOTAL TXs: %d\n", len(blockRes.Block.Txs)))
```

Comment:
Just as a totally minor nit, this would likely be more appropriate as a `bytes.Buffer` -- we are dynamically building these strings, so they may as well be `fmt.Fprintf(b, "...")` to keep everything as byte slices internally.

Reply:
Ha! I had it as a `bytes.Buffer` at first. I'll make the change.